src/issue-details.ts

```diff
@@ -0,0 +1,197 @@
+export interface IssueDetails {
+  id: string
+  number: number
+  title: string
+  body: string
+  state: string
+  created_at: string
+  updated_at: string
+  closed_at: string | null
+  comments: number
+  reactions: ReactionsInfo
+  comments_data: CommentData[]
+  user: UserInfo
+  assignees: UserInfo[]
+}
+
+export interface CommentData {
+  id: number
+  user: UserInfo
+  created_at: string
+  reactions: ReactionsInfo
+}
+
+export interface ReactionsInfo {
```

Comment:
I think this should maybe be ReactionData with these prooperties


    string id
    UserInfo user
    string reaction
    string created_at